fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -833,6 +834,9 @@ public boolean getUseScanBlockCache() {
     @VarAttr(name = ENABLE_RULE_BASED_MATERIALIZED_VIEW_REWRITE)
     private boolean enableRuleBasedMaterializedViewRewrite = true;
 
+    @VarAttr(name = ANALYZE_FOR_MV)
+    private String analyzeForMV = "sample";
+
```

Comment:
Why not keep the same logic as the olap table?

Reply:
I'm afraid that's too expensive for MV,